templates/error.html

```diff
@@ -0,0 +1,418 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Sivustolla on häiriö</title>
+    <style>
+      body {
+        font-family: Arial, sans-serif;
+        background-color: #f0f0f0;
+        text-align: center;
+        margin: 0;
+        padding: 0;
+      }
+
+      header {
+        display: flex;
+        height: 88px;
+        flex-direction: column;
+        justify-content: center;
+        align-items: flex-start;
+        align-self: stretch;
+        background: #fff;
+        width: 100%;
+        border-bottom: 1px solid #ccc;
+
+        .site-title {
+          display: flex;
+          padding: 16px 0px;
```

Comment:
Could we use common HDS CSS variables instead of pixel values? Not imported, just copied to this file. 

This is just my opinion, because it would look nicer in my old HDS eyes 😄 

Base values should be set first:
```
body {
    font-size: var(--fontsize-body-m);
    color: var(--color-black-90);
    line-height: var(--lineheight-l);
}
```

And then use vars instead of pixels. 
```
--breakpoint-m: 768px;
--container-width-xl: 1200px;

--color-white: #fff;
--color-black-90: #1a1a1a;
    
--spacing-xs: 0.75rem; /* 12px */
--spacing-s: 1rem; /* 16px */
--spacing-m: 1.5rem; /* 24px */
--spacing-l: 2rem; /* 32px */
--spacing-xl: 2.5rem; /* 40px */
--spacing-2-xl: 3.0rem; /* 48px */
--spacing-3-xl: 3.5rem; /* 56px */
--spacing-4-xl: 4rem; /* 64px */
--spacing-5-xl: 4.5rem; /* 72px */

--fontsize-heading-xxl: 4rem; /* 64px */
--fontsize-heading-xl: 3rem; /* 48px */
--fontsize-heading-xl-mobile: 2.5rem; /* 40px */
--fontsize-heading-l: 2rem; /* 32px */
--fontsize-heading-m: 1.5rem; /* 24px */
--fontsize-heading-s: 1.25rem; /* 20px */
--fontsize-heading-xs: 1.125rem; /* 18px */
--fontsize-heading-xxs: 1rem; /* 16px */

--fontsize-body-s: 0.875rem; /* 14px */
--fontsize-body-m: 1rem; /* 16px */
  ```
  
  PS. looks like`#f0f0f0` is not found in HDS colors.

Reply:
Added variables. I didn't even remember that those are a thing in plain css.